.travis.yml

```diff
@@ -60,7 +60,7 @@ before_script:
     # Get the test cases
     - git clone https://github.com/su2code/TestCases.git ./TestData
     - cd TestData/
-    - git checkout develop
+    - git checkout feature_bsplineFFD
```

Comment:
You're probably still working on this one.. but don't forget to switch this back to develop.